src/hb-common.h

```diff
@@ -171,7 +275,171 @@ HB_EXTERN hb_language_t
 hb_language_get_default (void);
 
 
-/* hb_script_t */
+/**
+ * hb_script_t:
+ * @HB_SCRIPT_COMMON: HB_TAG ('Z','y','y','y')
+ * @HB_SCRIPT_INHERITED: HB_TAG ('Z','i','n','h')
+ * @HB_SCRIPT_UNKNOWN: HB_TAG ('Z','z','z','z')
+ * @HB_SCRIPT_ARABIC
```

Comment:
Perhaps not. I thought that the non-obvious ones might be wanting an explanation, but that if we didn't include the whole list, it would be more confusing. I also considered copying in the HB_TAGs as commentary for each entry, but decided it was not urgent.